dataactcore/models/stagingModels.py

```diff
@@ -1694,6 +1694,7 @@ def __init__(self, **kwargs):
 Index("ix_pafa_uri_upper", func.upper(PublishedAwardFinancialAssistance.uri))
 Index("ix_pafa_awarding_subtier_c_upper", func.upper(PublishedAwardFinancialAssistance.awarding_sub_tier_agency_c))
 Index("ix_pafa_afa_generated_unique_upper", func.upper(PublishedAwardFinancialAssistance.afa_generated_unique))
+Index("ix_pafa_record_type", func.upper(PublishedAwardFinancialAssistance.record_type))
```

Comment:
1) Why are we uppering the record type? Why not just set `index=True` in the column definition?
2) It is ALREADY defined that way and this index already exists

Reply:
Force of habit of UPPERCASE'ing everything and confirmed there are no letter record_types in the data. Removing.